movement/validators/files.py

```diff
@@ -339,6 +324,62 @@ def _file_contains_valid_frame_numbers(self, attribute, value):
                 "file. ",
             )
 
+    def _extract_frame_numbers_from_file_attributes(
+        self, df, file_attributes_dicts
+    ):
+        """Get frame numbers from the 'frame' key under 'file_attributes'."""
+        list_frame_numbers = []
+        for k_i, k in enumerate(file_attributes_dicts):
+            try:
+                list_frame_numbers.append(int(k["frame"]))
+            except Exception as e:
```

Comment:
Can this be constrained to catch specifically a ValueError? Or can int() also throw other types of errors?
This so thread my help https://stackoverflow.com/questions/8075877/converting-string-to-int-using-try-except-in-python